app/views/api/metric_visibilities/change.js.erb

```diff
@@ -0,0 +1,19 @@
+<%# If EventTarget is replaceWith'ed immediately, ajax:success and ajax:complete are not called and the spinner will stay %>
+setTimeout(() => {
+  const oldRow = document.getElementById('metric_<%= @metric.id %>');
+  const newRow = '<%= escape_javascript(render(partial: "api/metrics/metric", locals: { metric: @metric })) %>';
```

Comment:
@josemigallas so, what's the difference between 
```
render(partial: "some/path", locals: { object: @object })
```
and
```
render("some/path", object: @object)
```
? in some cases one is used, and in other cases another one.

Reply:
The latter is a shorthand of the former, but they are the same.
Long form `render partial: ''` is unnecessary unless using a layout.
I tend to use it though.. because I like explicit, but I admit it's unnecessary verbose.